web/ui/react-app/src/pages/Config.tsx

```diff
@@ -1,6 +1,46 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import { RouteComponentProps } from '@reach/router';
+import { Alert, Button } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import CopyToClipboard from 'react-copy-to-clipboard';
 
-const Config: FC<RouteComponentProps> = () => <div>Config page</div>
+import './Config.css';
+
+const Config: FC<RouteComponentProps> = () => {
+  const [config, setConfig] = useState(null);
+  const [error, setError] = useState("");
+  const [copied, setCopied] = useState(false);
+
+  useEffect(() => {
+    fetch('../api/v1/status/config')
+      .then(res => res.json())
+      .then(res => setConfig(res.data.yaml))
+      .catch(error => setError(error.message));
+  });
+
+  return (
+    <>
+      <h2>
+        Configuration&nbsp;
+        <CopyToClipboard
+          text={config ? config! : ''}
+          onCopy={() => { setCopied(true); setTimeout(() => setCopied(false), 1500);}}
+        >
+          <Button color="light" disabled={!config}>
+            {copied ? 'Copied' : 'Copy to clipboard'}
+          </Button>
+        </CopyToClipboard>
+      </h2>
+
+      {error
+        ? <Alert color="danger"><strong>Error:</strong> Error fetching configuration: {error}</Alert>
+        : config
+          ? <pre className="config-yaml">{config !== null ? config : "Loading..."}</pre>
```

Comment:
Hm, "config" should always be !== null, right?

Reply:
It's set to `null` via `useState()` before it's loaded (and the spinner works).

Reply:
I mean this statement ```config !== null ? config : "Loading...``` - it should always be different than null, otherwise will not reach this line
``` config // is not null -> let's render <pre className=.....>{ here config cannot be null, so it always render config and never  will render 'Loading' }</pre>```

Reply:
Ahhh, sorry, I totally forgot to delete that part after adding the spinner. Oops!